doc/nrf/ug_thread_configuring.rst

```diff
@@ -198,7 +208,10 @@ OpenThread configuration sets
 *****************************
 
 |NCS| provides predefined sets of features and optional functionalities from the OpenThread stack.
-You can use these sets for configuration purposes or for building libraries with complete Thread specification support.
+These predefined sets are based on :ref:`nrfxlib:ot_libs` available in nrfxlib.
```

Comment:
I would rather say that :ref:`nrfxlib:ot_libs` provide functionalities listed in these predefined sets.